2024/wg-fedid.html

```diff
@@ -167,17 +167,26 @@ <h2>Motivation and Background</h2>
     <section id="scope" class="scope">
       <h2>Scope</h2>
       <p>
-        The Working Group will specify new web platform features intended to be implemented in user agents like browsers. The purpose of these features is to support privacy-preserving authentication, authorization flows, and requesting federated identities without compromising security principles for Identity Providers (IdPs) or Relying Parties (RPs) (in a ‘traditional’ federation model) or Issuers, Verifiers, and Holders (in a digital identity wallet architecture), and User Agents. Here, “privacy” minimally refers to the appropriate processing of personal information and preventing third parties from gleaming anything about the end-user’s environment (e.g., which wallets are available and their capabilities). This work results in developing new mechanisms that define how information is passed by the browser between the different entities and authentication intermediaries to facilitate federated authentication; these mechanisms are not authentication methods.
+        The Federated Identity Working Group defines Web Platform features that allow user agents to support secure and privacy-preserving interactions related to digital identities. 
       </p>  
+      <p>
+        These features are intended to support different interaction flows (e.g., authentication, authorization, requesting identities or credentials, and issuance) in a 'traditional' federated identity model - with Identity Providers (IdPs) or Relying Parties (RPs) - and in a digital wallet 'decentralized' model - with Issuers, Holders, and Verifiers.
       <p>
         If any mechanisms developed to support authentication and authorization flows would cause breaking changes for existing protocols, work on that mechanism must include a well-documented transition period.
       </p>
-
-      <section id="section-out-of-scope">
-        <h3 id="out-of-scope">Out of Scope</h3>
-        <p>
-          The identity space is much larger than that of federated authentication and digital credential wallets. While several topics related to identity may be of interest, they are out of the scope for our work.
-        </p>
+      <p>
+        Here, 'privacy' minimally refers to the appropriate processing of personal information and preventing third parties from learning anything about the end-user's environment (e.g., which wallets are available, their brand, and their capabilities). 
```

Comment:
There are some cases where the identity system necessarily involves the RP learning something about the end-user's environment (like, in some scenarios, which wallet they had installed and used to present a credential). Maybe just clarify slightly? 

```suggestion
        Here, 'privacy' minimally refers to the appropriate processing of personal information and preventing third parties from unnecessarily learning anything about the end-user's environment (e.g., which wallets are available, their brand, and their capabilities). 
```